compiler/tests/schema/namespace_basic_types.bond

```diff
@@ -0,0 +1,6 @@
+ namespace tests2
```

Comment:
The gRPC commit would be easier to review if you first committed the change to to the test schema with the updated golden files.